Changelog.md

```diff
@@ -8,10 +8,10 @@
 - Update Camel Quarkus Catalog from 1.8.1 to 2.0.0
 - Adapt for Camel K community 1.5/Camel K Red Hat 1.4
   - Provide completion for new Camel K modeline option names
-  - Provide completion for the different kinds of `config` Camel K modeline option (`configmap`, `secret`, `file`) 
+  - Provide completion for the different kinds of `config` and `resource` Camel K modeline option (`configmap`, `secret`, `file`) 

```

Comment:
I think it should be "options" in this case because its config and resource options